CMakeLists.txt

```diff
@@ -13,6 +13,11 @@ project(iCub-Tests)
 # and with yarp support enabled
 find_package(RTF 1.0.1 COMPONENTS YARP REQUIRED)
 
+# setting default compilation to release/optmized
+if(NOT CMAKE_BUILD_TYPE)
+	set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the type of build, recommanded options are: Debug or Release" FORCE)
```

Comment:
Not a real issue, but this will add the CMAKE_BUILD_TYPE on multi-config generators (i.e. MSVC and XCode) where this variable is not used...
This is cleaner:

```cmake
if(NOT CMAKE_CONFIGURATION_TYPES)
  if(NOT CMAKE_BUILD_TYPE)
    ...
  endif()
endif()